freezeyt/freezer.py

```diff
@@ -59,11 +59,14 @@ async def freeze_async(app, config):
 }
 
 
-def check_mimetype(url_path, headers, default='application/octet-stream'):
+def check_mimetype(
+    url_path, headers,
+    default='application/octet-stream', *, get_mimetype=guess_type,
+):
     if url_path.endswith('/'):
         # Directories get saved as index.html
         url_path = 'index.html'
-    file_type, file_encoding = guess_type(url_path)
+    file_type, _ = get_mimetype(url_path)
```

Comment:
How to handle our expectations about returning just a string but `guess_type` returning a `Tuple` of `strings`